src/data/list/basic.lean

```diff
@@ -1526,6 +1526,14 @@ by rw [h, prod_append, prod_cons, mul_left_comm]; exact dvd_mul_right _ _
 @[simp] theorem sum_const_nat (m n : ℕ) : sum (list.repeat m n) = m * n :=
 by induction n; [refl, simp only [*, repeat_succ, sum_cons, nat.mul_succ, add_comm]]
 
+theorem dvd_sum [comm_semiring α] {a} {l : list α} (h : ∀ x ∈ l, a ∣ x) : a ∣ l.sum :=
+begin
```

Comment:
I would have expected a term-mode pattern matching proof from Chris... I'm happy to see you are coming over to the light side of the force. [/joking]